src/Core/src/Platform/Android/ActivityExtensions.cs

```diff
@@ -0,0 +1,49 @@
+using AndroidX.Window.Layout;
+using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui.Platform
+{
+	internal static partial class ActivityExtensions
+	{
+		internal static IWindow? GetWindowFromActivity(this Android.App.Activity? activity)
```

Comment:
Deleting this and moving this code to `ContextExtensions.GetWindow` which we are already using

![image](https://user-images.githubusercontent.com/5375137/190507543-1ec523b7-f3d7-4651-9de8-680917fea981.png)

Reply:
Beautiful. Less code is good code.